package.json

```diff
@@ -73,6 +72,7 @@
     "through2": "^2.0.3",
     "tinyify": "^2.4.0",
     "v8-compile-cache": "^1.1.0",
+    "wasm-brotli": "^1.0.1",
```

Comment:
@goto-bus-stop can you try upgrading to `>= 1.0.2`? That will have the fix for 32-bit targeted rust crates.

Reply:
I just tried this, and it's working great! Yay :sparkles: